packages/ui/src/Avatar.tsx

```diff
@@ -37,12 +39,34 @@ interface AvatarProps {
   /**
    * The URL of the user's image.
    */
-  src?: string;
+  imageSrc?: string;
```

Comment:
Any reason to change this from src?

Reply:
Honestly, it was just to make it more explicit but I have no strong opinion one way or the other.

Reply:
I would lean towards keeping the prop name and not breaking the API whenever possible.

Reply:
You got it. I'll switch it back